.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I think.. I'm going to keep it in for the moment. If there are complaints, we can review it later.